source/exercises/taper.md

```diff
@@ -1,2 +1,98 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.0
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
 # 波形尖灭
 
+- 本节贡献者: {{何星辰}}、{{田冬冬}}、{{姚家园}}
+- 最近更新日期: 2025-11-07
+- 预计花费时间: 20 分钟
+
+---
+波形尖灭是指在信号首尾施加平滑衰减窗函数，使振幅在两端平滑过渡至零，从而减少截断引起的频谱泄漏。设原始信号为 $x(j)$、窗函数为 $w(j)$，则尖灭后的信号可表示为：
+
+$$
+x'(j) = w(j)*x(j),
+$$
+
+其中 $w(j)$ 在信号首尾逐渐从 0 过渡到 1，并在中间区域保持为 1，实现波形的尖灭。
+
```

Comment:
The image `source/exercises/taper_functions.png` should be displayed in this tutorial. 

Instead of inserting the static image, we can use the `code-cell` block to generate the image dynamically.